allennlp/data/token_indexers/token_indexer.py

```diff
@@ -48,9 +47,10 @@ def get_padding_token(self) -> TokenType:
 
     def get_padding_lengths(self, token: TokenType) -> Dict[str, int]:
         """
-        This method returns a padding dictionary for the given token.  For single ID tokens, e.g.,
-        this dictionary will be empty, but for a token characters representation, this will return
-        the number of characters in the token.
+        This method returns a padding dictionary for the given token which provides additional
```

Comment:
Instead of "which provides additional information ...", I'd say "that specifies lengths for all arrays that need padding."